### ISSUE TITLE
Failed to install assimp

### ISSUE THREAD

**Issue by HardGaming01** [microsoft/vcpkg#6737]
These are the output messages
```

λ vcpkg.exe install assimp
The following packages will be built and installed:
    assimp[core]:x86-windows
Starting package 1/1: assimp:x86-windows
Building package assimp[core]:x86-windows...
-- Using cached C:/CXXDev/vcpkg/downloads/assimp-assimp-v4.1.0.tar.gz
-- Using source at C:/CXXDev/vcpkg/buildtrees/assimp/src/v4.1.0-e8080a98e2
-- Configuring x86-windows
-- Building x86-windows-dbg
CMake Error at scripts/cmake/vcpkg_build_cmake.cmake:202 (message):
    Command failed: C:/msys64/mingw32/bin/cmake.exe;--build;.;--config;Debug;--target;install;--;-v
    Working Directory: C:/CXXDev/vcpkg/buildtrees/assimp/x86-windows-dbg
    See logs for more information:
      C:\CXXDev\vcpkg\buildtrees\assimp\install-x86-windows-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/assimp/portfile.cmake:33 (vcpkg_install_cmake)
  scripts/ports.cmake:71 (include)


Error: Building package assimp:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: assimp:x86-windows
  Vcpkg version: 2018.11.23-nohash
```

This is the log file produce
[install-x86-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/3247016/install-x86-windows-dbg-out.log)

I've installed several different packages including boost, qt5 and etc. Assimp is the only one failed to install.

**Comment by HardGaming01** [comment]
I just noticed that vcpkg is calling the CMake built within msys2 so I moved the custom version of CMake up in environment PATH and it built successfully. 

However when I try to link the assimp in CLion it failed with 
`LINK : fatal error LNK1104: cannot open file 'assimp-vc140-mt.lib'`

Which seem to be associated with the bug I had when building assimp, at the end of the output log I posted above there is such a line
`LINK : fatal error LNK1104: cannot open file 'minizip.lib'`
and in both situations I've checked that the .lib files are sitting right under
/vcpkg/installed/x86-windows/lib/ directory.

However when I tried to build the program in VS2019 with the same CMake configuration it seem to build and run successfully. I wonder if there's any difference running a CMake project from Visual Studio than directly dealing with it.
Other packages like SDL2 and SDL2-image built with vcpkg can be linked within CLion with no problem.

I've tried to run CMake as administrator but that didn't work either

**Comment by Voskrese** [comment]
`vcpkg owns minizip | findstr lib`

`vcpkg owns assimp | findstr lib`

**Comment by HardGaming01** [comment]
> `vcpkg owns minizip | findstr lib`

This is the output when I try vcpkg owns

```
C:\CXXDev\vcpkg (master -> origin)
λ vcpkg owns minizip | findstr lib
minizip:x86-windows: x86-windows/debug/lib/minizip.lib
minizip:x86-windows: x86-windows/lib/minizip.lib
minizip:x86-windows: x86-windows/tools/minizip/zlib1.dll

C:\CXXDev\vcpkg (master -> origin)
λ vcpkg owns assimp | findstr lib
assimp:x86-windows: x86-windows/debug/lib/assimp-vc140-mtd.lib
assimp:x86-windows: x86-windows/debug/lib/pkgconfig/assimp.pc
assimp:x86-windows: x86-windows/lib/assimp-vc140-mt.lib
assimp:x86-windows: x86-windows/lib/pkgconfig/assimp.pc
```

So Assimp is already built and runs pretty well in VS2019 but the only thing bothering me right now is why it's not running correctly in CLion with the linker error 
"LINK : fatal error LNK1104: cannot open file 'assimp-vc140-mt.lib'"
while the library is right there and can be used correctly by visual studio

**Comment by Voskrese** [comment]
**debug**
debug/lib/assimp-vc140-mtd.lib mt**d**.lib

**Comment by HardGaming01** [comment]
Thank you, I see the problem now.
I just renamed the library file and deleted the 'd' suffix, cleared the CMake cache and now the linker tells me it's looking for "assimp-vc140-mtd.lib". And after I change that library file name back everything starts working correctly.

I guess it's because of the previous CMake cache that's telling the linker to look for "assimp-vc140-mt.lib" because I configured the cmake project before I properly install assimp

Thank you
